lib/rules/web/ocm_console/ocm_overview_page.xyaml

```diff
@@ -0,0 +1,39 @@
+--- # the rules for the ocm overview page
+check_overview_nav_item:
+  element: &overview_nav_item
+    selector:
+      xpath: //nav/ul/li/a[@href='https://qaprodauth.cloud.redhat.com/openshift/overview']
+go_to_ocm_overview_page:
+  element:
+    <<: *overview_nav_item
+    op: click
+check_empty_overview_page:
```

Comment:
Too many elements in the overview rule. Advise you separate them in different blocks, etc. according to the separation in the page. This makes us easily maintain the code in future.